SUMMARY.md

```diff
@@ -4,13 +4,13 @@
 
 - [Getting started](doc/guide/getting-started.md)
   - [Reloading the RP2040](doc/guide/rp2040-update.md)
-  - [Reloading the FPGA Image](doc/guide/fpga-update.md)
-  - [Building Examples](doc/guide/building-examples.md)
```

Comment:
I prefer the previous style. Is there a particular English style lowRISC follows?

Reply:
I'm used to sentence case in academic writing because it makes it easier to read headings. Either way it was inconsistent.